src/util.js

```diff
@@ -13,3 +13,13 @@ export const objectFromArray = (arr, createValue) =>
 export const mapObject = (fn, obj) => Object.keys(obj).reduce(
   (acc, key) => ({ ...acc, [key]: fn(obj[key], key, obj) })
 , {});
+
```

Comment:
yeah, but from the function name and the parameters you can guess what's happening ;)